transactional-event-api/src/main/java/com/github/jonasrutishauser/transactional/event/api/monitoring/ProcessingFailedEvent.java

```diff
@@ -15,4 +17,30 @@ public Exception getCause() {
         return cause;
     }
 
+    @Override
+    public String toString() {
+        return "ProcessingFailedEvent [cause=" + cause + ", getEventId()=" + getEventId() + ", toString()="
+                + super.toString() + ", hashCode()=" + hashCode() + ", getClass()=" + getClass() + "]";
+    }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = super.hashCode();
+        result = prime * result + Objects.hash(cause);
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
+            return true;
+        if (!super.equals(obj))
+            return false;
+        if (getClass() != obj.getClass())
+            return false;
```

Comment:
Dieser Check wird doch bereits in der Super Klasse gemacht.

Reply:
das passiert wenn man's einfach eclipse generieren lässt ohne zu reviewen ;-)